packages/atomic/src/components/facets/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -382,17 +382,32 @@ export class AtomicNumericFacet
     return this.shouldRenderInput || this.shouldRenderValues;
   }
 
+  private get hasInputRange() {
+    return !!this.filterState?.range;
+  }
+
   private get shouldRenderValues() {
-    const hasInputRange = !!this.filterState?.range;
-    return !hasInputRange && !!this.valuesToRender.length;
+    return !this.hasInputRange && !!this.valuesToRender.length;
   }
 
   private get shouldRenderInput() {
     if (!this.withInput) {
       return false;
     }
 
-    return this.searchStatusState.hasResults || !!this.filterState?.range;
+    if (this.hasInputRange) {
+      return true;
+    }
+
+    if (!this.searchStatusState.hasResults) {
+      return false;
+    }
+
+    if (!this.valuesToRender.length && this.numberOfValues > 0) {
```

Comment:
This one is not clear ... there are no values to render, and the number of values being requested is greater than 0, then do not render?

Why not just the first half of the condition?

Reply:
That's what I had put in the first place, but there's a very specific test in the cypress suite that verify that if you want to have a "just an input", you can set `number-of-values="0" with-input="true"`, and that the input should be displayed.

I think it makes sense to allow that....

@ThibodeauJF Do you remember exactly what was the context behind that feature ? Was it part of the mockups ?